invokeai/frontend/web/src/features/parameters/store/generationSlice.ts

```diff
@@ -87,7 +91,8 @@ export const initialGenerationState: GenerationState = {
   shouldUseNoiseSettings: false,
   steps: 50,
   threshold: 0,
-  tileSize: 32,
+  infillTileSize: 32,
+  infillPatchmatchDownscaleSize: 2,
```

Comment:
This yields terrible results for smaller image sizes eg 512x512. This should default to 1 and users should be advised that they can increase this to improve performance.

Reply:
Done on both.